WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -116,7 +116,16 @@ open class QuickStartTourGuide: NSObject {
         WPAnalytics.track(.quickStartSuggestionViewed)
     }
 
-    func start(tour: QuickStartTour, for blog: Blog) {
+    /// Starts the specified tour.
+    ///
+    /// - Parameters:
+    ///     - tour: the tour to start.
+    ///     - blog: the blog in which the tour will take place.
+    ///     - immeditately: whether the tour must start immediately or not.  If this is `false`
+    ///         the tour will be initialized but the first step won't be executed.  The caller of
+    ///         this method must also call `run()` to effectively start the tour.
+    ///
+    func start(tour: QuickStartTour, for blog: Blog, immediately: Bool = false) {
```

Comment:
Why add a parameter that is never used? Unless I'm mistaken, after this PR `immediately` is always false.

Reply:
Actually this was a mistake in my PR, as the default value was meant to be `true`.  I've just fixed it.

There are two calls to this method.  The one that delays running the tutorial:

https://github.com/wordpress-mobile/WordPress-iOS/blob/e3ee8cd321feb74e37f69d012346aff898a64484/WordPress/Classes/ViewRelated/Blog/QuickStartChecklistViewController.swift#L102-L107

And the one that I didn't want to modify, which was previously and still is immediate:

https://github.com/wordpress-mobile/WordPress-iOS/blob/e3ee8cd321feb74e37f69d012346aff898a64484/WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift#L104-L107

I could make the immediate one still make both calls if we prefer, but I wanted to bring up both scenarios are currently used.  Thoughts?

Reply:
I'd prefer that both use the same process to start a tour, as it keeps the API simpler. I'd like to break up `start(tour:blog:)` into two methods. The immediate start version can call both methods in sequence.

I propose:
`start(tour:blog:)` become `prepare(tour:blog:)`
`run()` become `begin()`

Reply:
Changed the process so that it's now the same for both cases.